src/cryptoadvance/specter/hwi_server.py

```diff
@@ -6,12 +6,15 @@
 
 rand = random.randint(0, 1e32) # to force style refresh
 
-hwi = HWIBridge()
+hwi = None
 
 @app.route("/hwi_rpc/", methods=["POST"])
 @login_required
 def api():
     """JSON-RPC for ... anything. In this case - HWI Bridge"""
+    global hwi
```

Comment:
maybe makes sense to create hwi when we create specter?
Like in `Specter.__init__`:
```py
class Specter:
    def __init__(self, data_folder="./data", config={}):
        self.hwi = HWIBridge(self)
        #...
```

Then we can access it as `app.specter.hwi`

Reply:
I was thinking to make them just completely separated like this: https://github.com/cryptoadvance/specter-desktop/pull/143#discussion_r432836087
What do you think?